FirebasePerformance/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased
```

Comment:
I'm going to update the changelogs now. In case this makes it in before code freeze, I'm suggesting the correct changelog header.

```suggestion
# 10.27.0
```